src/objects.js

```diff
@@ -1,42 +1,54 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {
+    name: name,
+    age: age
+  };
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
-const hasProperty = (property, object) => {
-  // your code here
-};
+function hasProperty(property, object) {
+  return object. hasOwnProperty (property);
+}
 
 const isOver65 = person => {
-  // your code here
+  return person.age > 65;
 };
 
 const getAges = people => {
-  // your code here
+  let result = people.map(a => a.age);
+  return result;
+  //worked but don't understand how the function inside .map's brackets really works
```

Comment:
Could be made into a one liner:
```suggestion
  return people.map(person => person.age);
```
If you look at the test, this is expecting an array of all ages of the people in the array provided. `.map` creates an array so thats part of the problem solved, internally it returns a value calculated for each value in the array it is 'map'ing over (represented by the placeholder character `a` here, you could call this whatever you want.)
Since each element in the array is an object, you are saying that for each `person` object you want to return the age property, which map adds to its array. Hope that makes more sense!